doc/BUILD-Debian.md

```diff
@@ -0,0 +1,74 @@
+1. install boost (apt-get)
+2. install NVidia drivers
+3. install CUDA
+4. install ccmake
+5. install OpenCV
+6. install eigen
+7. install jpeg9a
+8. install libpng
+9. install ncurses
+10. install CMake
+11. create the machine-specific config file
+12. compile CCTag
+
+1. install boost (apt-get)
+apt install boost
+
+2. Install NVidia drivers
+Download the latest drivers from NVidia. Allow them to disable nouveau.
+Reboot. Install. Reboot.
+
+3. install CUDA
```

Comment:
This info is not up-to-date. AFAIK the latest CUDA version is 9. Version 7.5 is problematic with some cards, including GTX 980 Ti and GTX 1080.